src/departure/index.js

```diff
@@ -39,6 +43,10 @@ export function getDeparturesFromStopPlaces(
         timeRange = 72000,
         start = new Date(),
         includeNonBoarding = false,
+        whiteListed = {
```

Comment:
Kanskje det er eit enklare API å la konsumenten sende inn `whiteListedLines` og `whiteListedAuthorities`, i staden for eit objekt?

Reply:
Enig, ble litt ryddigere.